docs/source/llms/gateway/deprecation.rst

```diff
@@ -1,6 +1,100 @@
 .. _gateway-deprecation:
 
+=============================
 MLflow AI Gateway Deprecation
 =============================
 
-TODO
+The MLflow AI Gateway is deprecated and has been replaced by the MLflow deployments API.
+This page describes how to migrate from the MLflow AI Gateway to the MLflow deployments API.
+
```

Comment:
Can we have a brief callout that the "fluent API" is being removed and that a direct client access paradigm is required now (just so that users who are accustomed to the convenience of the fluent implementation aren't confused when they try to convert their fluent code to the deployments namespace and end up thinking that the implementation is broken)?

Reply:
Added the tables below.